src/components/DocWrapper/DocCallout/DocCallout.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import DocCallout, {DocCalloutProps, CalloutType} from '.';
+
+const baseProps: DocCalloutProps = {
+  className: 'Test',
+  type: CalloutType.note,
+};
+
+describe('DocCallout Component', () => {
+  it('renders without crashing', () => {
+    render(<DocCallout {...baseProps} />);
+
+    expect(screen.getByTestId('DocCallout')).toBeTruthy();
+  });
+
+  it('renders with default className', () => {
+    render(<DocCallout {...baseProps} />);
+
+    expect(screen.getByTestId('DocCallout')).toHaveClass('DocCallout');
+    expect(screen.getByTestId('DocCallout__header')).toHaveClass('DocCallout__header');
```

Comment:
Could you also add a check for the className with modifier (`DocCallout${modifier}`, `DocCallout__header${modifier}`) here?

Reply:
Yes, sure